psalm.xml.dist

```diff
@@ -611,6 +611,9 @@
             <errorLevel type="suppress">
                 <!-- See https://github.com/vimeo/psalm/issues/5472 -->
                 <file name="src/Portability/Converter.php"/>
+
+                <!-- See https://github.com/vimeo/psalm/issues/3399 -->
+                <file name="tests/Functional/PortabilityTest.php"/>
```

Comment:
Yes. Otherwise, we have this error:
```
ERROR: InvalidDocblock - tests/Functional/PortabilityTest.php:66:5 - :: in array key is only allowed for ::class in docblock for Doctrine\DBAL\Tests\Functional\PortabilityTest::caseProvider (see https://psalm.dev/008)
    /** @return iterable<string, array{ColumnCase::LOWER|ColumnCase::UPPER, list<string>}> */
    public static function caseProvider(): iterable
```

Reply:
But you don't use an array key here. Did you try to wrap the union into parenthesis, e.g. `array{(ColumnCase::LOWER|ColumnCase::UPPER), list<string>}`? That usually does the trick with Psalm.

Reply:
I wasn't aware of this. Thank you so much for the hint!
The error is fixed.